packages/base/card-api.gts

```diff
@@ -1252,17 +1252,17 @@ class LinksToMany<FieldT extends CardConstructor>
   }
 
   private async loadMissingFields(
-    instance: CardBase,
+    instance: Card,
     notLoaded: NotLoaded,
     identityContext: IdentityContext,
     relativeTo: URL | undefined
-  ): Promise<CardBase[]> {
+  ): Promise<Card[]> {
     let refs = (notLoaded.reference as string[]).map(
       (ref) => new URL(ref, instance.id ?? relativeTo).href // new instances may not yet have an ID, in that case fallback to the relativeTo
```

Comment:
perhaps relativeTo is not needed?

Reply:
we'll still need it. for new cards that don't have an ID assigned yet we'll need a relativeTo